javascript-primitives-and-variables/main.js

```diff
@@ -0,0 +1,18 @@
+var fullName;
+fullName = 'Junou Lee';
+var isCool = true;
+var totalPets;
+totalPets = 2;
+var unknown = null;
+var never;
```

Comment:
You can assign a value on the same line that you declare a variable, this is unnecessary

Reply:
![image](https://user-images.githubusercontent.com/117887912/207135986-307c56ab-9cda-4406-bba8-c0660b080b7a.png)